### PR TITLE
feat: add support for new Go stubs dir locations

### PR DESCRIPTION
**Pull request by codyoss**
- Be able to handle migrated proto stubs
- Default release level as beta so new apis generate warnings in godoc

Related: 490324477

### CODE REVIEW

**Comment by codyoss** [comment]
We will want to wait until the cloud dir in go-genproto is locked down before we merge this.

File: bazel/src/main/java/com/google/api/codegen/bazel/BazelBuildFileView.java [hunk 1]
```diff
@@ -175,12 +175,29 @@ class BazelBuildFileView {
     tokens.put("rest_numeric_enums", numericEnums);
   }
 
-  private String assembleGoImportPath(boolean isCloud, String protoPkg, String goPkg) {
+  /**
+   * Returns the import path for a Go GAPIC library package.
+   *
+   * @param isCloud is true if cloud is in the goP
```
Comment (noahdietz) [inline bazel/src/main/java/com/google/api/codegen/bazel/BazelBuildFileView.java:181]:
```suggestion
   * @param isCloud is true if cloud is in the goPkg
```

**Review by noahdietz — COMMENTED** [review]
Since we are forcing `release_level = "beta"`, at some point we will likely want to change it to `alpha` or `stable` or something. Can you add it to the list of "Preserved attributes" so that if there is a manual change to the value, we don't overwrite it? https://github.com/googleapis/rules_gapic/blob/main/bazel/src/main/java/com/google/api/codegen/bazel/ApiVersionedDir.java#L80

Just add it to that list.

**Comment by codyoss** [comment]
@noahdietz Ahh I missed that, thank you. Done.

**Review by noahdietz — APPROVED** [review]

**Pull request merged**